src/Arcus.BackgroundJobs.AzureAppConfiguration/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -65,7 +78,25 @@ public static class IServiceCollectionExtensions
                     .AddCloudEventBackgroundJob(subscriptionNamePrefix, serviceBusTopicConnectionStringSecretKey, configureBackgroundJob)
                     .WithServiceBusMessageHandler<RefreshAppConfigurationMessageHandler, CloudEvent>(
                         messageBodyFilter: message => message?.Type is "Microsoft.AppConfiguration.KeyValueModified" 
-                                                      || message?.Type is "Microsoft.AppConfiguration.KeyValueDeleted");
+                                                      || message?.Type is "Microsoft.AppConfiguration.KeyValueDeleted",
```

Comment:
> This is more then just adding remarks :)

It's what the title states and the issue suggest we do. It's closely related to each other since the dedicated initialization make the newly added remarks explicit.

> Was this auto refresh not working previously ?

No, it's only to make the initialization process more dev-friendly.